RLTsquare/CustomRoute/etc/frontend/routes.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
+    <router id="standard">
+        <route id="routing" frontName="routing">
```

Comment:
* Naming convention -  we should not use "routing" for id/frontName... we should adjust the naming properly to match our module.